cdap-ui/app/features/tracker/directives/top-entity-graph/top-entity-graph.js

```diff
@@ -161,24 +161,37 @@ angular.module(PKG.name + '.feature.tracker')
           let sidebarElem = angular.element(parentContainer[0]).find('div');
 
           angular.forEach(scope.model.results, (result, index) => {
+            let tooltipContent = result.entityName;
+            let topOffset = y(index) + margin.top + (y.rangeBand()/2);
+
             if (scope.type === 'applications') {
               scope.programsPath = 'apps.detail.overview.programs({ appId: "' + result.entityName + '" })';
-            }
-            if (scope.type === 'programs') {
+            } else if (scope.type === 'programs') {
               let programType = myLineageService.parseProgramType(result.programType);
               scope.programsPath = programType + '.detail.run({ appId: "' + result.application + '", programId: "' + result.entityName + '" })';
```

Comment:
We should probably just go to `programType.detail({...})` state. Right now we are going to runs state and since we don't have runid the breadcrumb looks off.